node/src/test_utils/database_utils.rs

```diff
@@ -40,6 +40,27 @@ pub fn bring_db_0_back_to_life_and_return_connection(db_path: &Path) -> Connecti
     conn
 }
 
+pub fn bring_db_9_back_to_life_and_return_connection(db_path: &Path) -> Connection {
```

Comment:
I cannot properly review code if there is some code missing. I haven't seen this used anywhere.